chall04/sadawi.py

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/python3
+#Python 3.6.9
+
+import sys
+
+def error(str):
+	print("Error: " + str)
+	sys.exit()
+
+try:
+	i = int(input())
+except:
+	error("Input is not a digit.")
+	sys.exit()
```

Comment:
error(), already sys.exit, so the following sys.exit() line 14 is not necessary.

error() is only used once, so I would avoid creating a function for that and you could use sys.exit("message") ; https://docs.python.org/2/library/sys.html#sys.exit